Adds the base for cargo bounties

Aranclanos opened the pull request:
### Purpose
Adds cargo bounties base, three different bounties for cargo techs to complete from roundstart.
Moves the CargoData SO and script to their correct folders.
Once arriving centcomm the cargo shuttle will check the contents of each wrapped objects for processing and ignore the wrapping object price values.
Adds the bounty window to the cargo console UI.
Reworks the cargo console UI, making it simpler and easier to understand and fixing some bugs when two consoles were used at the same time.
Adds the base for the emagging of the cargo console.

makes progress on #6150.

### Notes:
To check for completion status of the bounties, they'll check for itemattributes traits
The three base bounties are to get 5 wrenches, 5 welders and 5 crowbars. They reward 200 points.

Aranclanos committed:
Adds cargo bounties base, three different bounties for cargo techs to complete from roundstart.
Moves the CargoData SO and script to their correct folders.
Once arriving centcomm the cargo shuttle will check the contents of each wrapped objects for processing and ignore the wrapping object price values.
Adds the bounty window to the cargo console UI.
Reworks the cargo console UI, making it simpler and easier to understand and fixing some bugs when two consoles were used at the same time.
Adds the base for the emagging of the cargo console.

Aranclanos committed:
Fixes a codacy issue

corp-0 commented on UnityProject/Assets/Scripts/Managers/CargoManager.cs:
```diff
@@ -303,18 +308,66 @@ public void DestroyItem(ObjectBehaviour item, HashSet<GameObject> alreadySold)
+			var itemAttributes = item.GetComponent<ItemAttributesV2>();
+			if (itemAttributes)
+			{
+				foreach (var itemTrait in itemAttributes.GetTraits())
```
can you reformulate this logic to reduce nesting? Maybe use ``Validations.HasItemTrait`` instead of iterating all traits or even extract some portions into private methods.

corp-0 commented on UnityProject/Assets/Scripts/Systems/Cargo/CargoData.cs:
```diff
@@ -6,8 +6,8 @@ namespace Systems.Cargo
 {
 	public class CargoData : ScriptableObject
 	{
-		[ReorderableList]
```
why did you remove the tooltip and reorderable attribute?

Aranclanos committed:
Lets the category list of CargoData be reorderable on the inspector again.
Reduces the nesting on the cargo shuttle arriving item check for bounties.
Saves the omitted information on the bounties OS and CargoData.

corp-0 commented on UnityProject/Assets/Scripts/Managers/CargoManager.cs:
```diff
@@ -223,11 +219,20 @@ public void OnShuttleArrival()
+			var wrappedItem = item.GetComponent<WrappedBase>();
+			if (wrappedItem)
+			{
+				var wrappedContents = wrappedItem.GetOrGenerateContent().GetComponent<ObjectBehaviour>();
```
this ObjectBehaviour can be null

Aranclanos committed:
Changes cargo shuttle item handling using the seemingly useless class ObjectBehaviour to PushPull.
